scripts/simplifiedUI/ui/simplifiedUI.js

```diff
@@ -457,9 +457,16 @@ function onGeometryChanged(rect) {
     }
 }
 
-function ensureFirstPersonCameraInHMD(isHMDMode) {
+var TIMEOUT_BEFORE_REHIDE_TOOLBAR_MS = 700
+function onDisplayModeChanged(isHMDMode) {
     if (isHMDMode) {
         Camera.setModeString("first person");
+    } else {
+        //works for now, but not a permanent fix by any means.
```

Comment:
```suggestion
        // works for now, but not a permanent fix by any means.
```

And while you're at it, please add this space here for style consistency!

Reply:
Ack

Reply:
Ack